### DISCUSSION TITLE
Please put all breaking changes in the changelogs

### DISCUSSION THREAD

**Discussion by LelouBil** [borgmatic-collective/docker-borgmatic#372] [General]
There's a warning in the README: Secrets will be implemented differently from October 2024. From \*\_FILE to FILE_\_\*

But nothing related to this in releases changelogs

**Comment by grantbevis** [comment]
https://github.com/borgmatic-collective/docker-borgmatic/releases/tag/1.9.0

It's in point 2 here

**Reply by LelouBil** [reply]
Could there be a specific "Breaking changes" section for future changelogs ?
